fixups/EnvVarFixup/EnvVarFixup.xml

```diff
@@ -3,7 +3,7 @@
          SchemaLocation="https://github.com/Microsoft/MSIX-PackageSupportFramework/blob/master/MetadataSchema.xml ..\MetadataSchema.xsd">
   >
   <Version>1.0.0</Version>
-  <MinimumWindowsPlatform>10.0.14393.0</MinimumWindowsPlatform>
+  <MinimumWindowsPlatform>10.0</MinimumWindowsPlatform>

```

Comment:
This should stay at 14393.0.  PSF guarantees an RS1+ compatibility (Current it is RS2+).  Moving this higher would break that promise.

Reply:
Same for every version change.

Reply:
Reverted this change